website/docs/docs/on-premises/releases/1.28.0.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 1.28.0
+---
+
+## Release date
+
+28 November 2023
+
+## Features
+
+* Add endpoint to list branches
+* Removed "generate project" links
+* Onprem customers can now specify additional sources to add to the Content-Security-Policy
```

Comment:
These notes are only for onprem, so you can remove the qualification at the start of the sentence. It should have a similar "voice" to the other features. eg.

"Allow additional sources to be added to the Content-Security-Policy"